xml/docu_styleguide.language.xml

```diff
@@ -405,6 +405,49 @@
    </para>
    </sect2>
 
+ <sect2 xml:id="sec-quotation">
+  <title>Quotations</title>
+  <para>
+  Use quotations to quote from sources, such as books.
```

Comment:
That would include citations, right?

Reply:
Yes, citations would go within `quote` elements, too.